crates/red_knot_python_semantic/resources/mdtest/call/dunder.md

```diff
@@ -0,0 +1,125 @@
+# Dunder calls
+
+## Introduction
+
+This test suite explains and documents how dunder methods are looked up and called. Throughout the
+document, we use `__getitem__` as an example, but the same principles apply to other dunder methods.
+
+Dunder methods are implicitly called when using certain syntax. For example, the index operator
```

Comment:
Interesting -- I would probably just call `[]` a "subscription" (Brett [does similarly](https://snarky.ca/unravelling-subscriptions-in-python/)) rather than an "operator" -- but subscriptions are indeed listed in the [operator precedence](https://docs.python.org/3/reference/expressions.html#operator-precedence) section of the language reference.